frontend/packages/shared/src/hooks/useUniqueKey.ts

```diff
@@ -0,0 +1,38 @@
+import { useRef } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { ArrayUtils } from '@studio/pure-functions';
+
+type UseUniqueKey = {
+  removeKey: (index: number) => void;
+  getUniqueKey: (index: number) => string;
+};
+
+export type UseUniqueKeyArgs = {
+  maxNumberOfKeys: number;
+};
+
+export const useUniqueKey = ({ maxNumberOfKeys }: UseUniqueKeyArgs): UseUniqueKey => {
```

Comment:
Jeg ville kalt denne `useUniqueKeys` (i flertall) for å tydeliggjøre at den håndterer en _liste_ med nøkler.

Reply:
Fixed! 😊